build.gradle

```diff
@@ -37,6 +37,11 @@ buildscript {
     dependencies {
         classpath "${opensearch_group}.gradle:build-tools:${opensearch_version}"
         classpath "gradle.plugin.com.dorongold.plugins:task-tree:1.5"
+        configurations.all {
+            resolutionStrategy {
+                force("org.eclipse.platform:org.eclipse.core.runtime:3.29.0") // for spotless transitive dependency CVE (for 3.26.100)
```

Comment:
You have fix this in https://github.com/opensearch-project/ml-commons/pull/2378/files , why add this again?

Reply:
this branch was not rebased with main, let me try rebase